app/docs/dev/policies/meshratelimit.md

```diff
@@ -0,0 +1,184 @@
+---
+title: MeshRateLimit (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it shouldn't be mixed with [Rate Limit](../rate-limit).
+{% endwarning %}
+
+This policy enables per-instance service request limiting. Policy supports ratelimiting of HTTP/HTTP2 requests and TCP connections.
+
+The `MeshRateLimit` policy leverages Envoy's [local rate limiting](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter) for HTTP/HTTP2 and [local rate limit filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/local_rate_limit_filter) for TCP connections.
+
+You can configure:
+*  how many requests are allowed in a specified time period, and how the service responds when the limit is reached for HTTP/HTTP2.
+*  how many connections are allowed in a specified time period for TCP.
+
+The policy is applied per service instance. This means that if a service `backend` has 3 instances rate limited to 100 requests per second, the overall service is rate limited to 300 requests per second.
+
+{% warning %}
+{{site.mesh_product_name}} currently doesn't allow to configure rate limiting for the specific service
```

Comment:
This is a bit confusing to me, did you mean "from a specific service"? Looking at the targetRef support matrix it seems like we can only target `Mesh` in `from` so then the above would make sense. Also nitpick: missing a dot at the end.

Reply:
Do we even want this warning if we already have a targetRef matrix?